setup.py

```diff
@@ -26,6 +26,7 @@
       package_data={"dateutil.zoneinfo": ["dateutil-zoneinfo.tar.gz"]},
       zip_safe=True,
       requires=["six"],
+      tests_require=["freezegun"],
```

Comment:
I should note that I'm not sure that this is the most foolproof way of installing test-specific dependencies, as the only thing that cares about this particular line is, AFAIK, `python setup.py test`, and if we're going to move to `pytest`, we'll either want our own `setup.py` test runner that uses `pytest`, and/or we should have a separate script that declares and installs the test dependencies independent of `python setup.py test`. Not sure what most people do.